### ISSUE TITLE
Add option to allow (by default) 80/132 switching

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#196]
```
0.6.1-1 disabled 80/132 switching by default.  Most Xterms have allow132 label 
for their VT options.
```

Original issue reported on code.google.com by `dh.darre...@gmail.com` on 7 Jul 2010 at 4:11

**Comment by mintty** [comment]
Added option `Enable132ColumnSwitching`.

**Comment by mintty** [comment]
Released 2.9.3.
